.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -490,28 +490,23 @@ rm ${HWCRON_CRONTAB_FILE}
 # COMPILE AND INSTALL ANALYSIS TOOLS
 
 echo -e "Compile and install analysis tools"
+
 pushd ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools
-#git pull origin master
-make
 
-# copy the necessary files out of the repo
-mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
-mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/bin
-mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/lang
-#mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/config
-#rsync -rtz ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools/bin/count_node      ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/bin
-#rsync -rtz ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools/bin/count_token     ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/bin
-#rsync -rtz ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools/bin/count_function  ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/bin
-rsync -rtz ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools/lang/*              ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/lang
-#rsync -rtz ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools/config/*            ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/config
-rsync -rtz ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT_AnalysisTools/bin/*               ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools/bin
+# compile the tools
+stack --allow-different-user --no-terminal --install-ghc --copy-bins build
+
+popd
+
+rm -rf ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
```

Comment:
Let's use an rsync (with or without the delete option) to minimize copying in the re-install case.
The install with 'clean' can be used to delete then re-copy the files if something has gone weird.